nsiqcppstyle_exe.py

```diff
@@ -44,7 +44,7 @@
 except NameError:
     from sets import Set as set
```

Comment:
Also not needed

Reply:
For the record, the "sets" module is deprecated in Python 3, and the "set" built-in function should be used instead.  Since the above code imported sets.Set() and assigned it the "set" alias, we do not need to alter code that refers to "set".

I removed the selected try/except both from nsiqcppstyle_exe.py and from RULE_3_2_F_use_representitive_classname_for_cpp_filename.py.

Done.